<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTreeView" name="dataPPView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="KTabWidget" name="ktabwidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="selectTab">
       <attribute name="title">
        <string>Select Resources</string>
       </attribute>
       <widget class="KPushButton" name="startButton">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>30</y>
          <width>81</width>
          <height>91</height>
         </rect>
        </property>
       </widget>
       <widget class="Nepomuk::ResourceSearchWidget" name="resWidget" native="true">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>361</width>
          <height>351</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>371</width>
          <height>30</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Threshold:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KDoubleNumInput" name="thresholdNumInput">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.000100000000000</double>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="sliderEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>371</width>
          <height>25</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Nepomuk Uri</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KLineEdit" name="uriLineEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="settingsPage">
       <attribute name="title">
        <string>DataPP Settings</string>
       </attribute>
       <widget class="QLabel" name="sourceNameLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>401</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>No source</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDoubleNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Nepomuk::ResourceSearchWidget</class>
   <extends>QWidget</extends>
   <header location="global">resourcesearchwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
